acceptance/apps_env_test.go

```diff
@@ -68,9 +68,7 @@ var _ = Describe("apps env", func() {
 
 			It("is not present in the pushed workload", func() {
 				appDir := "../assets/sample-app"
-				out, err := env.Epinio(appDir, "apps", "push",
-					"--name", appName)
-				env.OnStageFailureShowStagingLogs(err, out, appName)
+				out, err := env.Epinio(appDir, appName)
```

Comment:
Should this be `EpinioPushDir` ?

Reply:
According to @jimmykarily there is no staging log for container image pushes, so I removed that.

Honestly, the way it works right now (if an error happens, look for staging in the error message), it might just be easier to always check the error in the old `env.Epinio` and not add any new helpers.